lib/shortener-default.js

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018, Compiler Explorer Team
+// Copyright (c) 2012, Matt Godbolt
```

Comment:
Can you change the copyright to 2019, Compiler Explorer Team (and the same below too). Thanks!